cmd/s2i/main.go

```diff
@@ -83,6 +83,13 @@ func newCmdBuild(cfg *api.Config) *cobra.Command {
 		Use:   "build <source> <image> [<tag>]",
 		Short: "Build a new image",
 		Long:  "Build a new Docker image named <tag> (if provided) from a source repository and base image.",
+		Example: `
+# Build an application Docker image from a Git repository
+$ s2i build git://github.com/openshift/ruby-hello-world openshift/ruby-22-centos7 hello-world-app
```

Comment:
Image streams have been updated, you reviewed the pr... :)

https://github.com/openshift/origin/blob/master/examples/image-streams/image-streams-centos7.json#L49

It's not the default (ie ref'd as latest) yet because technically SCL
hasn't GAed SCL 2.0 yet.

As for test scripts/other refs, we should at some point but again until the
images are finalized I'm reluctant.

Ben Parees | OpenShift
On Oct 30, 2015 5:45 AM, "Maciej Szulik" <notifications@github.com> wrote:

> In cmd/s2i/main.go
> <https://github.com/openshift/source-to-image/pull/322#discussion_r43485542>
> :
>
> > @@ -83,6 +83,13 @@ func newCmdBuild(cfg *api.Config) *cobra.Command {
> >  		Use:   "build <source> <image> [<tag>]",
> >  		Short: "Build a new image",
> >  		Long:  "Build a new Docker image named <tag> (if provided) from a source repository and base image.",
> > +		Example: `
> > +# Build an application Docker image from a Git repository
> > +$ s2i build git://github.com/openshift/ruby-hello-world openshift/ruby-22-centos7 hello-world-app
>
> Shouldn't we then update our test scripts and most importantly origin's
> examples/image-streams/image-streams-centos7.json ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/source-to-image/pull/322/files#r43485542>.
>